webpack.common.js

```diff
@@ -1,4 +1,5 @@
 const path = require('path');
+const { InjectManifest } = require('workbox-webpack-plugin');
```

Comment:
Linter will complain about space around braces.

Reply:
Will see about getting that better integrated. This feedback should be automated.